### ISSUE TITLE
What is the exact solution of the equation?

### ISSUE THREAD

**Issue by Ruoyan-Zhao** [EnigmaHuang/Poisson2D_FEM#1]

**Comment by EnigmaHuang** [comment]
I assume that your "exact" solution means the analytical solution. Usually, most PDEs do not have analytical solutions. You can try to solve the example questions with Mathematica. They may or may not have analytical solutions. Or, you can construct your own problem, define the ground true function u first, and then create a PDE.

**Comment by Ruoyan-Zhao** [comment]
感谢您的详细回答，您的意思是您博客中的f和g是直接构造出来的而不是根据精确解u构造出来的对吗？（如果是根据u构造出来的，那么这个u是什么呢？） 赵若琰 邮箱：ruoyan.zh@gmail.com 签名由 网易邮箱大师 定制 On 01/12/2021 22:01, 黄归一 wrote: I assume that your "exact" solution means the analytical solution. Usually, most PDEs do not have analytical solutions. You can try to solve the example questions with Mathematica. They may or may not have analytical solutions. Or, you can construct your own problem, define the ground true function u first, and then create a PDE. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by EnigmaHuang** [comment]
Yes, I directly and casually constructed the f and g functions instead of constructing a u function in advance.
